tests/view/renderer.js

```diff
@@ -918,26 +920,6 @@ describe( 'Renderer', () => {
 			expect( domSelection.getRangeAt( 0 ).collapsed ).to.equal( true );
 		} );
 
-		it( 'should not add ranges if different editable is selected', () => {
```

Comment:
You are right that manual test is missing, and unit test can be done too, checking whether selection is properly converted to backward selection.

However in the case of commented test it's prefectly valid to remove it, as I kinda removed the functionallity here. Before, theoretically, the code supported rendering mutli-range view selection to DOM. This test checked on of the cases of that scenario. Since this scenario is now gone (now only view selection anchor and focus is rendered) the test is incorrect.